lib/builder/builder.js

```diff
@@ -60,28 +60,9 @@ export default class Builder {
     this._buildStatus = STATUS.INITIAL
 
     // Stop watching on nuxt.close()
-    this.nuxt.hook('close', () => this.unwatch())
-  }
-
-  unwatch() {
-    if (this.filesWatcher) {
-      this.filesWatcher.close()
-    }
-
-    if (this.customFilesWatcher) {
-      this.customFilesWatcher.close()
+    if (this.options.dev) {
+      this.nuxt.hook('close', async () => { await this.unwatch() })
```

Comment:
We can simplyfy this to `() => this.unwatch()` (And preventing to create a wrapper Promise)

Reply:
yeah, that's good advice, will change it.